app/Services/MovieService.php

```diff
@@ -3,6 +3,8 @@
 interface MovieService {
     public function getAllMoviesByPage($page, $perPage, $genres);
 
+    public function storeOMDBMovie($movie);
```

Comment:
Ne treba da pravis metodu kao ovu, barem ne naziv `storOMDBMovie`. BE ne treba da interesuje odakle dolaze podaci sa fornta(odakle ih je on cupao, to je FE zaduzenje, a ne i BE). Ono sto tvoj BE interesuje da samo kreiras film kod njega i to je to. Odakle ? To ga ne intereujse . Cimaj ako je nejasno